src/App.tsx

```diff
@@ -1,18 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [total] = useState(items.length);
```

Comment:
you don't need the state here. you can even move this variable out of `App` after `items` variable
```suggestion
  const total = items.length;
```